peakina/readers/csv.py

```diff
@@ -22,24 +22,34 @@ def read_csv(
     # extra `peakina` reader kwargs
     preview_offset: int = 0,
     preview_nrows: Optional[int] = None,
+    # We're keeping keep_default_na as True to follow
+    # the same behaviour for typing on default pandas
+    keep_default_na: bool = True,  # pandas default: `True`
     # change of default values
-    keep_default_na: bool = False,  # pandas default: `True`
     error_bad_lines: bool = False,  # pandas default: `True`
     **kwargs: Any,
 ) -> pd.DataFrame:
     """
     The read_csv method is able to make a preview by reading on chunks
     """
     if preview_nrows is not None or preview_offset:
+        """
+        We're passing 'na_filter' as False on the preview only because it optimise the reading on
+        large csv file, cf pandas doc :
+        https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html#:~:text=na_filter%3DFalse%20can%20improve%20the%20performance%20of%20reading%20a%20large%20file
```

Comment:
They say it's optimizing the reading _for files that do not have any NA values_.
Maybe it's ok to never display n/a in the preview, but we should be aware that it c'll create a difference between what we see in the preview and what we get afterwards. Do you think it's worth the performance optimization?

Reply:
I don't think we should optimize unless we actually have performance issues

Reply:
⚖️⚖️⚖️... it may be confusing for some clients during their preview, that's true... let's keep the default value as True